src/index.html

```diff
@@ -17,11 +17,449 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              class="is-active menu__link"
+              href="apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              data-qa="nav-hover"
+              href="laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="video"
```

Comment:
The `href` attribute of the `a` tag should contain a valid URL. If you don't have a specific link to put here, you can use `href="#"` as a placeholder.